src/color.rs

```diff
@@ -254,6 +256,8 @@ impl Colors {
         m.insert(Elem::Links { valid: true }, Colour::Fixed(13));
         m.insert(Elem::Links { valid: false }, Colour::Fixed(245));
 
+        // TODO add this after we can use file to configure theme
+        // m.insert(Elem::TreeEdge, Colour::Fixed(44)); // DarkTurquoise
```

Comment:
Wanted to do this a little while ago, I think we can make the color that is returned an `Option` and make this return `None`? I think we can take that up in another PR as well. This would let us remove the `#[allow(dead_code)]` above the struct definition.

Reply:
https://github.com/Peltoche/lsd/issues/506 issue created